app/controllers/api/storage.php

```diff
@@ -581,10 +581,10 @@
                         ->setAttribute('mimeType', $mimeType)
                         ->setAttribute('sizeActual', $sizeActual)
                         ->setAttribute('algorithm', $algorithm)
-                        ->setAttribute('openSSLVersion', $openSSLVersion)
-                        ->setAttribute('openSSLCipher', $openSSLCipher)
-                        ->setAttribute('openSSLTag', $openSSLTag)
-                        ->setAttribute('openSSLIV', $openSSLIV)
+                        ->setAttribute('openSSLVersion', $openSSLVersion ?? null)
+                        ->setAttribute('openSSLCipher', $openSSLCipher ?? null)
+                        ->setAttribute('openSSLTag', $openSSLTag ?? null)
+                        ->setAttribute('openSSLIV', $openSSLIV ?? null)
```

Comment:
Do we really need null safety here? I dont think there is a different if we do `$openSSLVersion ?? null` vs just `$openSSLVersion`

Reply:
Without the null safety here we get a warning logged when we run tests since we are accessing a undefined value